packages/openneuro-components/src/search-page/SearchResultItem.tsx

```diff
@@ -251,8 +250,7 @@ export const SearchResultItem = ({
     <Tooltip
       tooltip="Visable to all viewers"
       flow="up"
-      className="result-icon result-publlic-icon"
-    >
+      className="result-icon result-publlic-icon">
```

Comment:
Just noticing this is "publlic" (two `l`s). I'm not finding anywhere else in the repo that uses "result-publlic-icon" or "result-public-icon", so unclear if this is a typo (and if so, if it has any meaningful effect).

Reply:
Probably a class in the previous design that can just be removed.